pkg/network/tracer/tracer_test.go

```diff
@@ -990,7 +990,7 @@ func initTracerState(t testing.TB, tr *Tracer) {
 	require.NoError(t, err)
 }
 
-func getConnections(t *testing.T, tr *Tracer) *network.Connections {
+func getConnections(t require.TestingT, tr *Tracer) *network.Connections {
```

Comment:
Do you want this to be an AssertT? This is being called from an EventuallyWithT.

Reply:
(and thus the require.NoError two lines down is a problem)